src/core/dbus-cgroup.c

```diff
@@ -2188,7 +2188,7 @@ int bus_cgroup_set_property(
                                 c->restrict_network_interfaces_is_allow_list = is_allow_list;
 
                         STRV_FOREACH(s, l) {
-                                if (!ifname_valid(*s)) {
+                                if (!ifname_valid_full(*s, IFNAME_VALID_ALTERNATIVE)) {
```

Comment:
I'm unsure when this code path is relevant, I was not hitting it in my testing with a service. I found this by searching for the error message I got, if it doesn't make sense to also change the validation here I'm happy to drop this hunk.

Reply:
it is relevant, it's used when you use "systemd-run" rather than a drop-in unit file to configure the option